<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DmWindow</class>
 <widget class="QWidget" name="DmWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>671</width>
    <height>470</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QSpinBox" name="DMInput">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>20</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QTableWidget" name="DMView">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>230</y>
     <width>401</width>
     <height>221</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>401</width>
     <height>16777215</height>
    </size>
   </property>
   <attribute name="horizontalHeaderDefaultSectionSize">
    <number>150</number>
   </attribute>
  </widget>
  <widget class="QPushButton" name="ViewMore">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>View More</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="AddDMWindow">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>100</y>
     <width>301</width>
     <height>81</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="AddDM">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>190</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Add DM</string>
   </property>
  </widget>
  <widget class="QPushButton" name="SendDM">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>410</y>
     <width>91</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Send New DM</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Input">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>60</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Select</string>
   </property>
  </widget>
  <widget class="QPushButton" name="GoBack">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>10</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
